src/collective/listingviews/browser/views/basic.py

```diff
@@ -68,9 +68,10 @@ def retrieve_listing_items(self):
         are a large number of fields in the view. No need
         to wake up all the objects.
         """
-        path = self.context.getPhysicalPath()
-        path = "/".join(path)
-        items = self.context.portal_catalog(path={"query": path, "depth": 1})
+        items = self.context.getFolderContents()
+        # path = self.context.getPhysicalPath()
```

Comment:
If this is no longer needed, it should be either removed from the code or a comment explaining why it is being commented out added. Otherwise it is just confusing to anyone coming back to the code later